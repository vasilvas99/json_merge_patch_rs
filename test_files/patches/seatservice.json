{
    "name": "seatservice-example",
    "image": {
        "name": "ghcr.io/eclipse/kuksa.val.services/seat_service:v0.1.0"
    },
    "host_config": {
        "extra_hosts": [
            "databroker-host:host_ip"
        ],
        "port_mappings": [
            {
                "protocol": "tcp",
                "container_port": 50051,
                "host_ip": "localhost",
                "host_port": 30051,
                "host_port_end": 30051
            }
        ]
    },
    "config": {
        "env": [
            "VEHICLEDATABROKER_DAPR_APP_ID=databroker",
            "BROKER_ADDR=databroker-host:30555",
            "RUST_LOG=info",
            "vehicle_data_broker=info"
        ]
    }
}
