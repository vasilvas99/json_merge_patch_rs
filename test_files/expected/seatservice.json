{
    "id": "",
    "name": "seatservice-example",
    "image": {
        "name": "ghcr.io/eclipse/kuksa.val.services/seat_service:v0.1.0",
        "decrypt_config": null
    },
    "host_name": "",
    "domain_name": "",
    "resolv_conf_path": "",
    "hosts_path": "",
    "hostname_path": "",
    "mounts": [],
    "hooks": [],
    "host_config": {
        "devices": [],
        "network_mode": "bridge",
        "privileged": false,
        "restart_policy": {
            "maximum_retry_count": 0,
            "retry_timeout": 0,
            "type": "unless-stopped"
        },
        "runtime": "io.containerd.runc.v2",
        "extra_hosts": [
            "databroker-host:host_ip"
        ],
        "port_mappings": [
            {
              "protocol": "tcp",
              "container_port": 50051,
              "host_ip": "localhost",
              "host_port": 30051,
              "host_port_end": 30051
            }
        ],
        "log_config": {
            "driver_config": {
                "type": "json-file",
                "max_files": 2,
                "max_size": "1M",
                "root_dir": ""
            },
            "mode_config": {
                "mode": "blocking",
                "max_buffer_size": ""
            }
        },
        "resources": null
    },
    "io_config": {
        "attach_stderr": false,
        "attach_stdin": false,
        "attach_stdout": false,
        "open_stdin": false,
        "stdin_once": false,
        "tty": false
    },
    "config": {
        "env": [
           "VEHICLEDATABROKER_DAPR_APP_ID=databroker",
           "BROKER_ADDR=databroker-host:30555",
           "RUST_LOG=info",
           "vehicle_data_broker=info"
        ],
        "cmd": []
    },
    "network_settings": null,
    "state": {
        "pid": -1,
        "started_at": "",
        "error": "",
        "exit_code": 0,
        "finished_at": "",
        "exited": false,
        "dead": false,
        "restarting": false,
        "paused": false,
        "running": false,
        "status": "",
        "oom_killed": false
    },
    "created": "",
    "manually_stopped": false,
    "restart_count": 0
}